rules/CodingStyle/Rector/Encapsed/EncapsedStringsToSprintfRector.php

```diff
@@ -250,4 +264,9 @@ private function createString(string $value): String_
             'kind' => $kind,
         ]);
     }
+
+    private function containsASCIIChar(string $content): bool
```

Comment:
doesn't the return type do the inverse?

it returns true when nonASCII chars are contained?

Reply:
Could be. 

> The \x01 in your string represents the ASCII control character with a hexadecimal value of 01.

Should be probably `containsControlASCIIChar()` :+1:

Reply:
I created PR:

- https://github.com/rectorphp/rector-src/pull/6720

to rename it.